packages/modules/api-key/src/migrations/Migration20241205122700.ts

```diff
@@ -0,0 +1,32 @@
+import { Migration } from "@mikro-orm/migrations"
+
+export class Migration20241205122700 extends Migration {
+  async up(): Promise<void> {
+    this.addSql(
+      'alter table if exists "api_key" add column if not exists "deleted_at" timestamptz null;'
+    )
+    this.addSql(
+      'alter table if exists "api_key" alter column "type" type text using ("type"::text);'
```

Comment:
question: Looks like this is redundant, since it's part of the up and down?

Reply:
Yes, looks redundant, but again generated automatically by MikroORM after switching to DML